src/Components/Analyzers/src/Microsoft.AspNetCore.Components.Analyzers.csproj

```diff
@@ -6,7 +6,7 @@
     <NoPackageAnalysis>true</NoPackageAnalysis>
     <GenerateDocumentationFile>false</GenerateDocumentationFile>
     <Description>Roslyn analyzers for ASP.NET Core Components.</Description>
-    <IsShippingPackage>true</IsShippingPackage>
+    <IsPackable>true</IsPackable>
```

Comment:
Huh? Don't we want to ship this analyzer?

Reply:
Ooops, s/ship/pack/

Reply:
Nevermind. Both of these setting match the defaults and could be left out entirely.